### PR TITLE
class complete

### PR DESCRIPTION
**Pull request by nataliaruzhytska**

### CODE REVIEW

File: oop/homework.py [hunk 1]
```diff
@@ -1,278 +1,219 @@
...
 
     def eat(self, product):
-        pass
+        product = product.lower()
+        if product == "fodder":
+            self._increase_saturation_level(10)
+        if product == "apple":
+            self._increase_saturation_level(5)
+        if product == "milk":
+            self._increase_saturation_level(2)
```
Comment (alchemixXx) [inline oop/homework.py]: it's better to set up class variable wit h dict. In this dict count the fodder and energy. Here just check if the fodder in the dict and take the value

File: oop/homework.py [hunk 2]
```diff
@@ -1,278 +1,219 @@
...
 
-    * Redefine method _set_average_speed
-      if age less or eq 5 return 90
-      if age between 5 and 15(including) return 75
-      if age grosser 15(not including) return 40
+    def eat(self, product):
+        if product == "gazelle":
+            self._increase_saturation_level(30)
+        if product == "rabbit":
+            self._increase_saturation_level(15)
```
Comment (alchemixXx) [inline oop/homework.py]: the same that in Cat

File: oop/homework.py [hunk 3]
```diff
@@ -1,278 +1,219 @@
...
     def door_square(self):
-        pass
+        return self.width * self.height
 
-    def door_price(self):
-        pass
+    def door_price(self, material):
+        if material == 'wood':
+            return self.door_square() * self.wood_price
+        if material == 'metal':
```
Comment (alchemixXx) [inline oop/homework.py]: elif?

File: oop/homework.py [hunk 4]
```diff
@@ -1,278 +1,219 @@
...
+        self.__door = None
 
-    * Implement method get_room_square that returns the square of our room
-      (from walls_square divide windows and door square)
+    def create_wall(self, width, height):
 
-    """
+        if height <= 0 or width <= 0:
+            raise ValueError("Value must be > 0")
+        if len(self.__walls) == 4:
```
Comment (alchemixXx) [inline oop/homework.py]: perhaps > will be more suitable for general case?

**Review by alchemixXx — CHANGES_REQUESTED** [review]

File: oop/homework.py (hunk 2 above)
Comment (nataliaruzhytska) [reply oop/homework.py]: Done

File: oop/homework.py (hunk 3 above)
Comment (nataliaruzhytska) [reply oop/homework.py]: Done

File: oop/homework.py (hunk 1 above)
Comment (nataliaruzhytska) [reply oop/homework.py]: Done

File: oop/homework.py (hunk 4 above)
Comment (nataliaruzhytska) [reply oop/homework.py]: Done

**Review by alchemixXx — APPROVED** [review]

**Pull request merged**